### PR TITLE
GET currencies exchange rate added

### PR DESCRIPTION
**Pull request by alamirfan**

### CODE REVIEW

File: src/common/openapi.yaml [hunk 1]
```diff
@@ -41,5 +41,7 @@ paths:
     $ref: 'paths/common@places.yaml'
   /common/currency/list:
     $ref: 'paths/common@currencies.yaml'
+  /common/currency/exchange_rate:
```
Comment (kartikeyrajvaidya) [inline src/common/openapi.yaml:44]: it should not be `/common/currency/exchange_rate:` it should be `/common/exchange_rate:`

Comment (alamirfan) [reply src/common/openapi.yaml:44]: I think we should keep **/currency/** folder in between for better separation/understanding

Also list of currencies is already implemented with that only, even places too.
cc: @aditya999123

Comment (kartikeyrajvaidya) [reply src/common/openapi.yaml:44]: currency and exchange_rates both are different top level resources so should have different path after <role>/

Comment (alamirfan) [reply src/common/openapi.yaml:44]: At line number 29 of this file, exchange rates have been mentioned under currencies.
(How come exchange rates and currency both are different? The exchange rate is of currencies only)

Comment (alamirfan) [reply src/common/openapi.yaml:44]: Also get_exchange_rate() in platform-api is under api > currencies > common.py
and the currency_layer.py under core > services has both list currency and exchange_rates in same file

File: src/common/paths/common@currencies@exchange_rate.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        application/json:
+          schema:
+            type: object
+            properties:
+              data:
+                type: object
+                description: |
+                  Contains exchange rate
+                additionalProperties:
+                  type: string
```
Comment (aditya999123) [inline src/common/paths/common@currencies@exchange_rate.yaml:40]: check this
float or decimal or number

**Review by aditya999123 — APPROVED** [review]

**Pull request merged**